File: frontend/src/features/actividades/calculo_generacion_documento/models/calculo_generacion_documento.ts
export interface CalculoGeneracionDocumento {
  id_calculo_generacion_documento: number;
  id_expediente: number;
  tipo_propiedad: string | null;
  tipo_direccion: string | null;
  direccion: string | null;
  region: string | null;
  comuna: string | null;
  existe_rol_avaluo: boolean | null;
  rol_avaluo: string | null;
  valor_avaluo_pesos: number | null;
  revision_rol_propiedad: 'ROL_INCORRECTO' | 'ACEPTAR' | null;
  valor_uf_fecha_hoy: number | null;
  fecha_calculo: string | null;
  valor_uf_fecha_calculo: number | null;
  is_enviar_reparo: boolean;
  observaciones: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by: number | null;
  modified_date: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/calculo_generacion_documento.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class calculo_generacion_documento : base_auditoria
    {
        public long id_calculo_generacion_documento { get; set; }
        public long id_expediente { get; set; }
        public string? tipo_propiedad { get; set; }
        public string? tipo_direccion { get; set; }
        public string? direccion { get; set; }
        public string? region { get; set; }
        public string? comuna { get; set; }
        public string? rol_avaluo { get; set; }
        public string? revision_rol_propiedad { get; set; }
        public decimal? valor_uf_fecha_hoy { get; set; }
        public DateTime? fecha_calculo { get; set; }
        public decimal? valor_uf_fecha_calculo { get; set; }
        public bool is_enviar_reparo { get; set; }
        public string? observaciones { get; set; }
    }
}
